Functions/src/FunctionsDataInputValidation/InputDataValidation.java

```diff
@@ -0,0 +1,27 @@
+package FunctionsDataInputValidation;
+
+import java.util.Scanner;
+
+public class InputDataValidation {
+
+    public static void main(String[] args) {
+        int hour, min;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("What time is it?");
+        System.out.print("Hour ");
+        hour = sc.nextInt();
+        System.out.print("Minutes ");
+        min = sc.nextInt();
+        TimeValidation(hour, min);
+    }
+
+    private static void TimeValidation(int hour, int min) {
```

Comment:
Numele functiei putea fi mai inspirat, de exemplu: `isTimeValid`, `isValid`. Functia ar fi trebuit sa faca doar validarea datelor de intrare si sa returneze un rezultat boolean(true/false), iar afisarea mesajelor de timp corect/incorect sa se faca in metoda main ep baza rezultatului returnat de functie.